src/js/main.js

```diff
@@ -713,7 +713,7 @@ function generateFilename(prefix, suffix) {
             filename = CONFIG.flightControllerIdentifier + '_' + filename;
         }
         if(CONFIG.name && CONFIG.name.trim() !== '') {
-            filename = filename + '_' + CONFIG.name.trim().replace(' ', '_');
+            filename = filename + '_' + CONFIG.name.trim().replace(/[<>:"/\\|?* ]/g, '_');
```

Comment:
All of these characters are perfectly fine for file names in linux and probably MacOS, and most of them are fine in Windows as well. I don't think there is a point in arbitrarily mangling the name that a user has chosen.

Reply:
When you try and use one of the unloved characters windows complains about them 
in this order: `\ / : * ? " < > |` 

I'm actually not sure why existing code is set to replace the first space with an underscore. Either do them all or leave them alone.